### PR TITLE
Added previous implementation as well as new trial

### PR DESCRIPTION
**Pull request by 7ayushgupta**
I've added the functionality of streaming images on the previous GUI implementation. Currently, I tried out a new package I found on Github, it's flexible but needs some work.

### CODE REVIEW

**Comment by 7ayushgupta** [comment]
@ksakash Change the name of debug_layer?

**Comment by ksakash** [comment]
@7ayushgupta what to name it instead?

**Comment by ksakash** [comment]
Also I have fixed the Travis building, so take it seriously now. The current build failed!

File: debug_layer/anahita_gui/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Graphical User Interface for ROS
+## Humanoid Robotics, IITK
```
Comment (ksakash) [inline debug_layer/anahita_gui/README.md]: Humanoid Robotics!?

**Comment by ksakash** [comment]
@7ayushgupta can you add screenshots of the GUI?

**Pull request merged**